pkgs/tools/package-management/nixops/nixops-dns.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, stdenv
+, buildGoPackage
+, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "nixops-dns";
+  version = "HEAD";
+
+  goDeps = ./deps.nix;
```

Comment:
is `deps.nix` missing in the pr?
I can also vendor deps in nixops-dns repo so we don't need it here. Which way do you prefer?